sql: disable split queues for multitenant admin tests

rafiss opened the pull request:
These tests need exact control over replicas to test relocating ranges.
The recent change in ec14795 made it so split points are added to system
tables when applying the ExcludeDataFromBackup flag. This test doesn't
care about those splits, so we can just disable the queue.

fixes https://github.com/cockroachdb/cockroach/issues/120772
fixes https://github.com/cockroachdb/cockroach/issues/120847
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/120901)
<!-- Reviewable:end -->

fqazi reviewed (approved):
> Reviewed 28 of 28 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/120901)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @stevendanna)



<!-- Sent from Reviewable.io -->

rafiss committed:
sql: disable split queues for multitenant admin tests

These tests need exact control over replicas to test relocating ranges.
The recent change in ec14795 made it so split points are added to system
tables when applying the ExcludeDataFromBackup flag. This test doesn't
care about those splits, so we can just disable the queue.

Release note: None

fqazi reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 34 of 34 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/120901#-:-NtqcQoc6coDaMPKEf-i:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @stevendanna)



<!-- Sent from Reviewable.io -->

rafiss commented:
tftr!

bors r+
